zc_install/sql/install/zero_dates_cleanup.sql

```diff
@@ -80,15 +80,3 @@ UPDATE upgrade_exceptions SET errordate = NULL WHERE errordate < '0001-01-01';
 UPDATE zones_to_geo_zones SET last_modified = NULL WHERE last_modified < '0001-01-01';
 UPDATE zones_to_geo_zones SET date_added = '0001-01-01 00:00:00' WHERE date_added < '0001-01-01' and date_added is not null;
 
-UPDATE media_clips SET date_added = '0001-01-01 00:00:00' WHERE date_added < '0001-01-01' and date_added is not null;
```

Comment:
Shouldn't these remain as part of an upgrade at least? Lest the tables need this fix and get repurposed/revisited (ideally) in transition to this removal? Likely though this will be something to be addressed after the fact considering a void is being created without an alternative substitution first existing.